tools/android_lint/baseline.xml

```diff
@@ -188,4 +188,26 @@
             column="24"/>
     </issue>
 
+    <issue
+        id="Recycle"
+        message="This `SurfaceTexture` should be freed up after use with `#release()`"
+        errorLine1="    final SurfaceTexture surfaceTexture = new SurfaceTexture(0);"
+        errorLine2="                                          ~~~~~~~~~~~~~~~~~~~~~">
+        <location
+            file="../../../flutter/shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java"
+            line="97"
+            column="43"/>
+    </issue>
```

Comment:
do you know why it's showing this issue? Is it the change from `registerTexture(entry.id(), surfaceTexture);` to `registerTexture(entry.id(), entry.textureWrapper());` in FlutterRenderer.java?

Reply:
I think the linter runs on any region of the code with a diff, and this is the first change to this code since the linter was added.